mxp/constraints.lisp

```diff
@@ -117,9 +121,35 @@
                                         (vanishes! MTNTOP)
                                         (eq! MTNTOP 1))))))
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;                                           ;;
+;;  2.6 The S1NZNOMXPX and S2NZNOMXPX flags  ;;
+;;                                           ;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defconstraint setting-s1nznomp ()
```

Comment:
I would move the `(debug ...)` constraints (binary + counter-constancies) here so that it's at the same spot as in the spec.